src/pages/Integrations/Create/styling/cardselect.scss

```diff
@@ -0,0 +1,60 @@
+.src-c-wizard__tile {
+  width: 100%;
+
+  .src-c-wizard__icon {
+    height: 40px;
+  }
+
+  &.disabled {
+    .src-c-wizard__icon {
+      opacity: 0.5;
+    }
+  }
+
+  .redhat-icon {
+    height: 50px;
+  }
+}
+
+.src-c-wizard--step-text {
+  max-width: 536px;
+}
+
+.src-c-wizard__radio {
+  .pf-v5-c-radio:not(:last-child) {
+    margin-bottom: var(--pf-global--spacer--lg);
+  }
+}
+
+.src-c-wizard__conditional-radio {
+  .pf-v5-c-radio:not(:last-child) {
+    margin-bottom: var(--pf-global--spacer--sm);
+  }
+  .pf-v5-c-radio__input {
+    margin-top: 3px;
+  }
+}
+
+.src-c-wizard__rhel-mag-label {
+  position: relative;
+  top: -3px;
+}
+
+.src-c-wizard__rhel-desc-title {
+  color: initial;
+}
+
+.src-c-wizard__summary-description-list {
+  .pf-v5-c-description-list__group {
+    grid-template-columns: 150px 1fr;
+  }
+}
+
+.src-m-wizard-hide-me {
+  opacity: 0;
+}
+
+.src-c-wizard--switch-description {
+  color:  #6a6e73;
```

Comment:
@InsaneZein I'm not sure some of these styles are used - I can't find any instances in `notifications-frontend`. Also - if we want to keep them - we should update this color be: `var(--pf-global--Color--200)`.

I don't think this is blocking though.

Reply:
Notifications is using css in js functionality. Since this is just a copy of existing code I think we are fine-ish with keeping this as scss file. However the colors should be used from pf pallete.